src/ApiRequestPanel.js

```diff
@@ -164,14 +164,14 @@ export class ApiRequestPanel extends AmfHelperMixin(EventsTargetMixin(HeadersPar
   }
 
   _renderServerSelector() {
-    const { amf, hideServerSelector, selectedServerType, selectedServerValue, serversCount, hideCustomServer } = this;
-    const hidden = (serversCount && serversCount < 1) || !!hideServerSelector;
+    const { amf, selectedServerType, selectedServerValue, noCustomServer } = this;
     return html`<api-server-selector
-      .hidden=${hidden}
-      .hideCustom=${hideCustomServer}
+      ?hidden="${this._shouldHideServerSelector()}"
```

Comment:
Why is this a function? The render function should only represent the state. I will LGTM this but please, try avoid it in the future.